export const resetError = setError => {
  setTimeout(() => {
    setError("");
  }, 4000);
};
